WhaleSpotting/Controllers/WhaleSightingController.cs

```diff
@@ -27,4 +28,18 @@ public IActionResult GetById([FromRoute] int Id)
             return NotFound();
         }
     }
+
+    [HttpGet("allpending")]
+    public IActionResult GetPendingSightings()
+    {
+        List<WhaleSighting> whaleSightingsPendingList = _whaleSightingService.GetPendingSightings();
+        List<WhaleSightingResponse> whaleSightingsPendingListResponses = new List<WhaleSightingResponse>();
+
+        foreach (WhaleSighting ws in whaleSightingsPendingList)
+        {
+            var ourWsResponse = new WhaleSightingResponse(ws);
+            whaleSightingsPendingListResponses.Add(ourWsResponse);
+        }
```

Comment:
Can we return a `List<WhaleSightingResponse>` from the `_whaleSightingService.GetPendingSightings()` method so we don't need to have all this logic in the controller?